packages/core/src/index.ts

```diff
@@ -2,4 +2,9 @@ export { createConfig } from "@/config/config.js";
 export { createSchema } from "@/schema/schema.js";
 export type { Block, Log, Transaction } from "@/types/eth.js";
 export type { Virtual } from "@/types/virtual.js";
-export { type MergeAbis, mergeAbis } from "@ponder/utils";
+export {
+  type MergeAbis,
+  mergeAbis,
+  loadBalancedTransport,
+  rateLimitedTransport,
```

Comment:
Viem transport naming is pretty terse `http`, `fallback`. Maybe we should follow suit with `loadBalance`, `rateLimit`.